core/payment/examples/payment_api.rs

```diff
@@ -89,9 +92,9 @@ pub async fn start_dummy_driver() -> anyhow::Result<()> {
 
 pub async fn start_erc20_driver(
     db: &DbExecutor,
-    requestor_account: Box<EthAccount>,
+    requestor_account: Box<SecretKey>,
```

Comment:
No need to use `Box` here anymore. The only reason for doing that was that `EthAccount::load_or_generate` returned a `Box`.